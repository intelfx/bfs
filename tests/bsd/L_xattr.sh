invoke_bfs . -quit -xattr || skip
make_xattrs || skip
bfs_diff -L . -xattr
