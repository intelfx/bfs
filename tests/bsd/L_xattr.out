./link
./xattr
./xattr_2
